Library/Homebrew/missing_formula.rb

```diff
@@ -11,34 +11,35 @@ def reason(name, silent: false)
       def blacklisted_reason(name)
         case name.downcase
         when "gem", /^rubygems?$/ then <<~EOS
-          Homebrew provides gem via: `brew install ruby`.
+          macOS provides gem as part of Ruby. To install a newer version:
+            brew install ruby
         EOS
         when "tex", "tex-live", "texlive", "latex" then <<~EOS
-          Installing TeX from source is weird and gross, requires a lot of patches,
-          and only builds 32-bit (and thus can't use Homebrew dependencies)
+          There are three versions of MacTeX.
 
-          We recommend using a MacTeX distribution: https://www.tug.org/mactex/
-
-          You can install it with Homebrew Cask:
+          Full installation:
             brew cask install mactex
+
+          Full installation without bundled applications:
+            brew cask install mactex-no-gui
+
+          Minimal installation:
+            brew cask install basictex
         EOS
         when "pip" then <<~EOS
-          pip is part of the python formula, and can be installed with:
+          pip is part of the python formula:
             brew install python
-          However you will then have two Pythons installed on your Mac,
-          so alternatively you can install pip via the instructions at:
-            #{Formatter.url("https://pip.readthedocs.io/en/stable/installing/")}
         EOS
         when "pil" then <<~EOS
-          Instead of PIL, consider `pip2 install pillow`.
+          Instead of PIL, consider pillow:
+            pip2 install pillow
         EOS
         when "macruby" then <<~EOS
-          MacRuby is not packaged and is on an indefinite development hiatus.
-          You can read more about it at:
-            #{Formatter.url("https://github.com/MacRuby/MacRuby")}
+          macOS provides Ruby. To install a newer version:
```

Comment:
This suggests `macruby` is a replacement for `ruby`, which is not the case (it allowed using ruby to build OSX apps). To suggest any command, maybe it should be `brew cask install rubymotion`, since MacRuby calls it [its spiritual successor](https://github.com/MacRuby/MacRuby#macruby), and the developer [is generous with the license prices](https://twitter.com/amirrajan/status/1094988676050882561).

Reply:
Thanks; done!

Reply:
TIL, thanks @vitorgalvao!